CHANGELOG.md

```diff
@@ -24,6 +28,15 @@
 
   ...though the exact form differs by dialect.
 
+- Change to function syntax: `let f = x -> ...` See
+  https://github.com/PRQL/prql/blob/main/web/book/src/queries/functions.md
```

Comment:
Why point to GitHub rather than the Book directly?

Reply:
As interim documentation for this unreleased change, I preferred to point to the page where I was certain the information exists.

Reply:
OK — re https://github.com/PRQL/prql/pull/2613/files#r1204663635 — helpful but would be even more helpful in a final state!